packages/infra/lib/ihe-stack.ts

```diff
@@ -86,6 +86,55 @@ export class IHEStack extends Stack {
       },
     });
 
+    // Create lambdas
+    const iti38Lambda = createLambda({
+      stack: this,
+      name: "ITI38",
+      entry: "iti38",
+      layers: [lambdaLayers.shared],
+      envType: props.config.environmentType,
+      envVars: {
+        ...(props.config.lambdasSentryDSN ? { SENTRY_DSN: props.config.lambdasSentryDSN } : {}),
+      },
+      vpc: props.vpc,
+      alarmSnsAction: props.alarmAction,
+    });
+
+    const iti39Lambda = createLambda({
+      stack: this,
+      name: "ITI39",
+      entry: "iti39",
+      layers: [lambdaLayers.shared],
+      envType: props.config.environmentType,
+      envVars: {
+        ...(props.config.lambdasSentryDSN ? { SENTRY_DSN: props.config.lambdasSentryDSN } : {}),
+      },
+      vpc: props.vpc,
+      alarmSnsAction: props.alarmAction,
+    });
+
+    const iti55Lambda = createLambda({
+      stack: this,
+      name: "ITI55",
+      entry: "iti55",
+      layers: [lambdaLayers.shared],
+      envType: props.config.environmentType,
+      envVars: {
+        ...(props.config.lambdasSentryDSN ? { SENTRY_DSN: props.config.lambdasSentryDSN } : {}),
+      },
+      vpc: props.vpc,
+      alarmSnsAction: props.alarmAction,
+    });
+
+    // Create resources for each lambda directly under the API root
+    const iti38Resource = api.root.addResource("ITI38");
+    const iti39Resource = api.root.addResource("ITI39");
+    const iti55Resource = api.root.addResource("ITI55");
+
+    // Add methods for each resource
+    iti38Resource.addMethod("ANY", new apig.LambdaIntegration(iti38Lambda));
+    iti39Resource.addMethod("ANY", new apig.LambdaIntegration(iti39Lambda));
+    iti55Resource.addMethod("ANY", new apig.LambdaIntegration(iti55Lambda));
```

Comment:
Here and above, let's please create a function for each lambda. Grouping them like this might look good when writing it, but as we add more stuff it makes it harder to maintain since the code is really organized towards domain/business logic, not non-functional requirements. Example: think in 6 months time, we added a bunch of other stuff on this file and now we want to split it per lambda/resource - it will be much easier if we write a function per lambda/resource/use case.